features/dd-sdk-android-session-replay/api/dd-sdk-android-session-replay.api

```diff
@@ -58,8 +59,10 @@ public final class com/datadog/android/sessionreplay/SessionReplayConfiguration$
 	public synthetic fun <init> (FILkotlin/jvm/internal/DefaultConstructorMarker;)V
 	public final fun addExtensionSupport (Lcom/datadog/android/sessionreplay/ExtensionSupport;)Lcom/datadog/android/sessionreplay/SessionReplayConfiguration$Builder;
 	public final fun build ()Lcom/datadog/android/sessionreplay/SessionReplayConfiguration;
+	public final fun getLogger ()Lcom/datadog/android/api/InternalLogger;
 	public final fun setDynamicOptimizationEnabled (Z)Lcom/datadog/android/sessionreplay/SessionReplayConfiguration$Builder;
 	public final fun setImagePrivacy (Lcom/datadog/android/sessionreplay/ImagePrivacy;)Lcom/datadog/android/sessionreplay/SessionReplayConfiguration$Builder;
+	public final fun setLogger (Lcom/datadog/android/api/InternalLogger;)V
```

Comment:
probably needed to be re-generated, it shouldn't be public with the current API layout 🤔